c7n/reports/csvout.py

```diff
@@ -182,6 +185,14 @@ def __init__(self, resource_type, extra_fields=(), include_default_fields=True,
     def headers(self):
         return self.fields.keys()
 
+    def extract_raw(self, arr):
+        # Remove the datetime field in order to use as JSON
+        for record in arr:
+            for k in record.copy():
+                if k == "CustodianDate":
+                    del record[k]
```

Comment:
rather than removing the records, we can just just utils.dumps which will do datetime serialization for json on datetime